website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4643,6 +4648,13 @@ element in that window has been processed. The `accumulation_mode` parameter
 sets the window's **accumulation mode**.
 {{< /paragraph >}}
 
+{{< paragraph class="language-go" >}}
+You set the trigger(s) for a `PCollection` by passing in the `WindowTrigger` parameter
```

Comment:
When directly referring to code constructs for Go, prefer the way it shows up in the code, including the usual package short name. eg. `beam.WindowTrigger`, `beam.WindowInto`, `beam.AccumulationMode` etc.  (Since the PCollection type isn't being referred to in code, it doesn't get the package prefix.)